### PR TITLE
New yubicloud has a load balancer

### PR DESCRIPTION
**Pull request by nrw505**
The updated YubiCloud now does its own high availability and does not require parallel querying of several servers.

* Updates the defaults to just use api.yubico.com
* Adds retry logic, since we're not defaulting to 5 tries in parallel any more.
* A bit of drive-by cleanup
* Bump library version and prepare for new release

### CODE REVIEW

**Comment by nrw505** [comment]
OK, figuring out what's up with Travis is going to take a day or two.

**Comment by nrw505** [comment]
Sorted out the Travis failures. Ready to be reviewed now.

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,9 @@
+* Version 2.7 released 2020-02-??
```
Comment (klali) [inline NEWS]: put this as "unreleased"? until it's actually out?

File: README [hunk 2]
```diff
@@ -17,8 +17,8 @@ installed PEAR.  On Debian/Ubuntu systems:
 
 Install the component by invoking:
 
-  $ wget https://developers.yubico.com/php-yubico/Releases/Auth_Yubico-2.6.tgz
-  $ pear install Auth_Yubico-2.6.tgz
+  $ wget https://developers.yubico.com/php-yubico/Releases/Auth_Yubico-2.7.tgz
```
Comment (klali) [inline README]: we could replace the version here with latest (https://developers.yubico.com/php-yubico/Releases/Auth_Yubico-latest.tgz) and not have to update it again.

Comment (nrw505) [reply README]: :+1:

File: NEWS (hunk 1 above)
Comment (nrw505) [reply NEWS]: Sure

**Pull request merged**